primitives/ethereum-poa/src/lib.rs

```diff
@@ -344,6 +344,40 @@ pub fn public_to_address(public: &[u8; 64]) -> Address {
 	result
 }
 
+// TODO: There's got to be a better place for this
+pub enum RuntimeApiCalls {
+	GrandpaAuthorities,
+}
+
+#[cfg(feature = "std")]
+impl std::string::ToString for RuntimeApiCalls {
+	fn to_string(&self) -> String {
+		match self {
+			RuntimeApiCalls::GrandpaAuthorities => "GrandpaApi_grandpa_authorities",
+		}
+		.into()
+	}
+}
+
+// I want to be able to generate something like this directly from the decl_runtime_apis! macro
+pub enum EthereumHeadersApiCalls {
```

Comment:
I think this should stay private as it's only used internally by SubstrateClient, no?

We could simplify this to a bunch of consts instead if it's not part of the public API. Seems that currently we never really utilize the strong typing of that enum.

Reply:
That's a good point. I put it here in order to keep its proximity close to the `decl_runtime_apis!` call where these enums come from.